src/components/RightSidebar/Participants/Participant.vue

```diff
@@ -844,17 +844,19 @@ export default {
 			if (Object.prototype.hasOwnProperty.call(this.participant, 'statusMessage')) {
 				// We preloaded the status when via participants API
 				return {
-					status: this.participant.status || null,
+					status: this.participant.status || '',
 					message: this.participant.statusMessage || null,
 					icon: this.participant.statusIcon || null,
+					clearAt: this.participant.statusClearAt || null,
 				}
 			}
 			if (Object.prototype.hasOwnProperty.call(this.participant, 'status')) {
 				// We preloaded the status when via search API
 				return {
-					status: this.participant.status.status || null,
+					status: this.participant.status.status || '',
```

Comment:
NcAvatar expects `null` if there's nothing for `hasStatus`, so it's better to revert this